scripts/video-helper.js

```diff
@@ -97,18 +97,22 @@ export function addSoundcloudShowHandler(link) {
 }
 
 export function addPlayIcon(parent) {
-  const iconWrapper = createElement('div', { classes: 'video-icon-wrapper' });
-  const icon = createElement('i', { classes: ['fa', 'fa-play', 'video-icon'] });
-  iconWrapper.appendChild(icon);
-  parent.appendChild(iconWrapper);
+  const playButton = document.createRange().createContextualFragment(`
+    <span class="icon icon-play-video">
```

Comment:
The new icon affects the content which is out of the redesign scope.
Before: 
<img width="889" alt="obraz" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/125962117/bd2b2543-578b-49d8-b0b2-61c05f68de0d">
After: 
<img width="889" alt="obraz" src="https://github.com/Netcentric/vg-volvotrucks-us-rd/assets/125962117/af3073bb-426a-4bef-b989-1349257f466c">

Icons should be bigger no wit is just 22px.

Testing url: https://87-resource-gallery--vg-volvotrucks-us-rd--netcentric.hlx.page/trucks/vnr-electric/

Reply:
I think we are replacing font-icons with svg here. However the svg should be 48 * 48 and circle should be 40. 
Am I correct @cogniSyb ?